tests/unit/source_code/test_s3fs.py

```diff
@@ -0,0 +1,117 @@
+import pytest
+
+from databricks.labs.ucx.source_code.base import Advice, Deprecation
+from databricks.labs.ucx.source_code.s3fs import S3FSUsageLinter
+
+
+class TestDetectS3FS:
```

Comment:
Our convention for pytest is NOT to group tests into classes, so that we keep the nesting to minimum. I don't see a valid reason to break this convention in this particular file.

Reply:
class removed.